packages/block-library/src/template-part/editor.scss

```diff
@@ -1,14 +1,18 @@
 .block-editor-template-part__selection-modal {
+	z-index: z-index(".block-editor-template-part__selection-modal");
+
 	// To keep modal dimensions consistent as subsections are navigated, width
 	// and height are used instead of max-(width/height).
-	@include break-small() {
-		width: calc(100% - #{ $grid-unit-20 * 2 });
-		height: calc(100% - #{ $header-height * 2 });
-	}
-	@include break-medium() {
-		width: $break-medium - $grid-unit-20 * 2;
-	}
-	@include break-large() {
-		height: 70%;
+	.components-modal__frame {
```

Comment:
Why we need this change?

Reply:
The `block-editor-template-part__selection-modal` is now the overlay class name, so we can apply `z-index`. We don't need to apply modal dimensions to the overlay.